OloEngine/src/OloEngine/Core/BuildInfo.cpp

```
@@ -0,0 +1,52 @@
+
+    const char* GetGitDescribe()
+    {
+        return OLO_BUILD_GIT_DESCRIBE;
+    }
+
+    const char* GetBuildTimestamp()
+    {
+        return OLO_BUILD_TIMESTAMP;
+    }
+
+    std::string GetBuildId()
```

Cppcheck (2.21.0)

[style] The function 'GetBuildId' is never used.

(unusedFunction)